CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -835,6 +835,20 @@ public let DECL_NODES: [Node] = [
     kind: .extensionDecl,
     base: .decl,
     nameForDiagnostics: "extension",
+    documentation: """
+        An `extension` declaration.
+
+        An example of an extension is
```

Comment:
I think that other declarations have an `Example` section and it would be good to be consistent here.

```suggestion
        ### Example
```